### PR TITLE
Use access token for Guy's account

### PR DESCRIPTION
**Pull request by gtrepta**

### CODE REVIEW

File: jenkins/Dockerfile [hunk 1]
```diff
@@ -27,12 +27,15 @@ RUN    apt-get update                  \
             make                       \
             maven                      \
             netcat-openbsd             \
+            opam                       \
```
Comment (ehildenb) [inline jenkins/Dockerfile]: We shouldn't need `opam` I don't think?

**Comment by dwightguth** [comment]
Should we really be storing credentials in a git repo?

File: instructions.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+openjdk-11-jdk
+pandoc
+pkg-config
+python3
+rapidjson-dev
+software-properties-common
+time
+zip
+zlib1g-dev
+```
```
Comment (ehildenb) [inline instructions.md]: Double check that this list is only the things left in `Dockerfile`.

At least `opam` is still here.

**Comment by ehildenb** [comment]
@dwightguth these just credentials for testing firefly website, not for actually running it. It's only for the sandbox database.

**Comment by ehildenb** [comment]
And we need to update them frequently enough (for testing) that re-storing them on Jenkins every time is too much of a hassle.

**Review by ehildenb — APPROVED** [review]

**Pull request merged**